### PR TITLE
Yarn

### PR DESCRIPTION
**Pull request by nlordell**
:point_up: 

### Test Case

CI

### CODE REVIEW

**Comment by nlordell** [comment]
Note we save about a minute on CI builds with this:
- Yarn with cache: [0.86s](https://travis-ci.org/gnosis/dex-contracts/builds/626744351#L326)
- NPM with cache: [61.44s](https://travis-ci.org/gnosis/dex-contracts/builds/626731886#L313)

**Review by josojo — APPROVED** [review]
Nice, this is a big performance difference.

Could you please add the package-lock.json in the .gitignore file. I would be the first one submitting the package-lock.json with my next PR, just because I am used to use npm.

**Review by fleupold — APPROVED** [review]
Nice, this closes #326

**Review by anxolin — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -79,55 +79,55 @@ npx truffle compile
...
+# Place owl token order for the fee mechanism
 npx truffle exec scripts/stablex/place_order.js --accountId=0 --buyToken=$TOKEN_ID_DAI --sellToken=0 --minBuy=1000 --maxSell=1000 --validFor=20 --network rinkeby
 
 ```
 
 Then, the market order can be place, after switching to another account. Usually, this is expected to happen via the UI. If it will be done via the console, following commands can be used:
 
 ```sh
-//Deposit funds into exchange:
+# Deposit funds into exchange:
```
Comment (bh2smith) [inline README.md:120]: Thanks! For fixing this!

**Review by bh2smith — APPROVED** [review]
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia2.giphy.com%2Fmedia%2FAPYioVROaqXnO%2Fgiphy.gif&token=yarn" data-gitmeme-token="yarn"></a>

**Pull request merged**